src/lib/pages/frontsite/SelfCheckout/SelfCheckout.js

```diff
@@ -135,7 +142,14 @@ class SelfCheckout extends React.Component {
     return (
       <Container className="spaced" textAlign="center">
         <Header as="h1">SELF-CHECKOUT</Header>
-        <BarcodeScanner onBarcodeDetected={this.onBarcodeDetected} />
+        <Container>
+          <BarcodeScanner onBarcodeDetected={this.onBarcodeDetected} />
+          <Message warning compact attached="bottom">
+            If your browser isn't able to scan the barcode,
+            <br />
+            try using another browser.
```

Comment:
could you add a recommendation for Firefox (was it firefox with which it tends to work better ? or was it chrome?)